clients/cli/src/main/java/org/apache/gravitino/cli/commands/Command.java

```diff
@@ -168,14 +169,14 @@ protected <T extends GravitinoClientBase> Builder<T> constructClient(Builder<T>
    * @param <T> The type of entity.
    */
   protected <T> void output(T entity) {
-    if (outputFormat == null) {
+    if (listOptions.getOutputFormat() == null) {
```

Comment:
Can listOptions be null?

Reply:
The `listOptions` feature is not implemented at this stage.